src/MakeLocalizeFunction.js

```diff
@@ -10,25 +13,33 @@ function makeLocalizeFunction(localization, nested) {
 }
 
 /**
+ * Find the key if the key is a path expressed with dots
+ *
+ * e.g.
+ * Code: __("errors.connectionError")
+ * Lang: {"errors": {"connectionError": "There was an error connecting."}}
+ * New Code: "There was an error connecting."
+ *
+ * @param {object} localization
+ * @param {string} nestedKey The original key
  *
- * @param {object}  localization
- * @param {string}  string key
  * @returns {*}
  */
-function byString(object, string) {
+function byString(localization, nestedKey) {
```

Comment:
`nestedkey` => `key`

Reply:
key is a variable used in the function, I rather not param reassign if I dont have to. nestedKey makes sense from what the function does.